server/api/user/user.model.spec.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+var should = require('should');
+var app = require('../../app');
+var User = require('./user.model');
+
+var user = new User({
```

Comment:
what does this file do? Dont add code that is a template if we havent put our code inside it yet